client/app/lib/providers/computecontroller.coffee

```diff
@@ -1258,6 +1301,33 @@ module.exports = class ComputeController extends KDController
     else showError 'Failed to share credential'
 
 
+  removeClonedFromAttr: (stackTemplate, callback = kd.noop) ->
```

Comment:
This function clean the JComputeStack and JStackTemplate instance's config information.
And update instance and stores.
Do you want me to put this functionality to `JStackTemplate` just for clean `clonedFrom` and `needUpdate` attributes?